Docs: Prometheus metrics browser (#35152)
grafana/grafana: docs/sources/datasources/prometheus.md

| `Query type`       | `Range`, `Instant`, or `Both`. When running **Range query**, the result of the query is displayed in graph and table. Instant query returns only the latest value that Prometheus has scraped for the requested time series and it is displayed in the table. When **Both** is selected, both instant query and range query is run. Result of range query is displayed in graph and the result of instant query is displayed in the table. |
| `Exemplars`        | Run and show exemplars in the graph.                                                                                                                                                                                                                                                                                                                                                                                                       |

### Metrics browser

The metrics browser allows you to quickly find metrics and select relevant labels to build basic queries.
When you open the browser you will see all available metrics and labels.
If supported by your Prometheus, each metric will show its HELP and TYPE as a tooltip.

> When you select a metric, the browser will narrow down the available labels to show only the ones applicable to the metric.

Reviewer: ```suggestion
When you select a metric, the browser narrows down the available labels to show only the ones applicable to the metric.
```

> You can then select one or more labels for which the available label values will be shown in lists in the bottom section.
> You can select one or more values for each label to tighten your query scope.

Reviewer: ```suggestion
You can then select one or more labels for which the available label values are shown in lists in the bottom section.
Select one or more values for each label to tighten your query scope.
```

> Pro tip: If you do not remember a metric name to start with, you can also select a few labels first and find relevant label values. Once you select a few, the set of applicable metrics is reduced, making it easy to find the metric you were looking for.

Reviewer: ```suggestion
>**Note:** If you do not remember a metric name to start with, you can also select a few labels first, to narrow down the list and then find relevant label values. 
```

> All lists in the metrics browser have a search field above them to quickly filter for metrics or labels that match a certain string. Notice that the values section only has one search field. Its filtering applies to all labels to help you find values across labels once they have been selected, e.g., among your labels `app`, `job`, `job_name` only one might contain the value you are looking for.

Reviewer: ```suggestion
All lists in the metrics browser have a search field above them to quickly filter for metrics or labels that match a certain string. The values section only has one search field. Its filtering applies to all labels to help you find values across labels once they have been selected, for example, among your labels `app`, `job`, `job_name` only one might with the value you are looking for.
```

> Once you are happy with your query, you can click on "Use query" to run the query. The button "Use as rate query" will add a `rate(...)[$__interval]` around your query to help write queries for counter metrics.

Reviewer: ```suggestion
Once you are satisfied with your query, click "Use query" to run the query. The button "Use as rate query" adds a `rate(...)[$__interval]` around your query to help write queries for counter metrics.
```

The "Validate selector" button will check with Prometheus how many time series are available for that selector.

> Limitations: The metrics browser has a hard limit of 10,000 labels (keys) and 50,000 label values (including metric names). If Prometheus returns more results the browser will continue functioning, but the result sets will be cut off above those limits.

Reviewer: ```suggestion
#### Limitations 

The metrics browser has a hard limit of 10,000 labels (keys) and 50,000 label values (including metric names). If your Prometheus instance returns more results, the browser will continue functioning. However, the result sets will be cut off above those maximum limits.
```
